Developing-new-features-with-feature-gating.md

```diff
@@ -149,20 +149,20 @@ They allow us to hide features that are still in development/still being tested.
 
 ### When to use feature flags?
 
-Feature flags should be used when you are working on a feature whose scale makes it hard to implement it all at once and will need to be added incrementally/if the feature is likely to cause breakages. Essentially, feature flags are a must for featues that are not ready for production/fully-tested yet.
+Feature flags should be used when you are working on a feature whose scale makes it hard to implement all at once, or for more experimental features that are likely to cause breakages. Essentially, feature flags are a must for features that are not ready for production/fully-tested yet.
 
 ### How to use feature flags?
 
-Say you are working on a large scale user-facing feature that will take multiple PRs to be fully implemented. The following is the recommended way to use feature flags in such a scenario (*recommended implies that this is the way we use/have used feature flags in the past in Oppia. We haven't encountered a scenario where we required an alternate workflow. Unless there are compelling reasons, this is the development workflow we expect developers to follow when it comes to employing the use of feature flags*):
+Say you are working on a large scale user-facing feature that will take multiple PRs to fully implement. The following is the recommended way to use feature flags in such a scenario (*this is the development workflow we expect developers to follow when it comes to employing the use of feature flags*):
```

Comment:
I'd go even stronger: the second sentence could just be "In such cases, please use feature flags to gate your feature appropriately by carefully following the steps below:"